scripts/dependency_services/common.sh

```diff
@@ -34,7 +34,7 @@ $DOCKER_COMPOSE up -d
 
 if [ $? -ne 0 ]; then
     echo "Something goes wrong, please check $DOCKER_COMPOSE output"
-    return
+    exit
```

Comment:
Should we use `exit 1`?

Reply:
Because `exit` by default return the error code of the previous command.

OK, the previous command is `echo`. Let me specify a custom response code.